.github/workflows/test.yml

```diff
@@ -3,6 +3,33 @@ name: 'Test'
 on:
   [push, pull_request, workflow_dispatch]
 jobs:
+
+  nightly:
```

Comment:
should the change in `test.yml` be in a separate PR?

Reply:
removed